filters/openpolicyagent/opaauthorizerequest/opaauthorizerequest.go

```diff
@@ -124,7 +124,7 @@ func (f *opaAuthorizeRequestFilter) Request(fc filters.FilterContext) {
 	authzreq := envoy.AdaptToExtAuthRequest(req, f.opa.InstanceConfig().GetEnvoyMetadata(), f.envoyContextExtensions, rawBodyBytes)
 
 	start := time.Now()
-	result, err := f.opa.Eval(ctx, authzreq)
+	result, err := f.opa.Eval(ctx, fc.Tracer(), authzreq)
```

Comment:
better to pass `span`, then to pass the tracer. You do not need to get span from context within `opa.Eval`, or do I miss something?

Reply:
Yeah, sure we could just pass the span down to `Eval(...)` and get the span from the context in the Request function, if this is better.